src/lib.rs

```diff
@@ -431,23 +431,31 @@ impl fmt::Display for FMatchError {
             f: &mut fmt::Formatter,
             s: &str,
             lno_chars: usize,
-            mark_line: Option<usize>,
+            mark_line: usize,
         ) -> fmt::Result {
-            for (num, line) in s.lines().enumerate() {
-                let mark = match mark_line {
-                    Some(ml) if ml == num + 1 => ">> ",
-                    _ => "   ",
-                };
-                writeln!(f, "{}{:width$}: {}", mark, num + 1, line, width = lno_chars)?;
+            let mut num_lines = 0;
```

Comment:
isn't `num_lines` a mirror of `i` here?

Reply:
Ha, sort of, yes! Fixed in https://github.com/softdevteam/fm/pull/12/commits/7836a8b3aef84429fe5ccccbca9ca9a8b70d83f7.